cEpiabm/src/covidsim.cpp

```diff
@@ -25,6 +41,14 @@ namespace epiabm
     {
         return 1.0;
     }
+    
+    double Covidsim::CalcSpaceSusc(
+        Cell* /*cell*/,
+        Person* /*infectee*/,
+        unsigned short int )
+    {
+        return 0.2;
+    }
 
 
 } // namespace epiabm
```

Comment:
```suggestion
} // namespace epiabm

```
Seems to want newline characters at the end of files